service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionController.java

```diff
@@ -133,6 +139,7 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmitEvent(
         Map<String, Object> data = caseDetails.getData();
         data.put("dateAndTimeSubmitted", DateTimeFormatter.ISO_OFFSET_DATE_TIME.format(zonedDateTime));
         data.put("dateSubmitted", DateTimeFormatter.ISO_LOCAL_DATE.format(zonedDateTime));
+        data.put("sendToCtsc", setSendToCtsc());
```

Comment:
Are there going to be any other rules of sending to CTSC other than feature toggle? If not, then we shouldn't put it in case data, it's irrelevant to the case. We should rather put feature toggle in `NotificationHandler`. It also makes it harder to follow what's exactly feature toggled, because now it requires searching where does the `sendToCtsc` flag is used.

Reply:
The move to CTSC will happen on the 30th March. Cases submitted before this date will not move to CTSC and continue sending notifications to HMCTS admin rather than CTSC. We decided it was best to have a flag on the case itself, set at the point of case submission.

Reply:
Makes sense to me